docs/_includes/star-rating.html

```diff
@@ -15,12 +17,80 @@
     {% endfor %}
     </ul>
 
-    <p>Below is the code example for how to use the "star-rating-2-5" symbol. Please refer to the <a href="#icon">icon section</a> for full guidance on how to include and use SVG symbols.
+    <p>Below is the code example for how to use the "star-rating-2-5" symbol. Please refer to the <a href="#icon">icon section</a> for full guidance on how to include and use SVG symbols.</p>
 
     {% highlight html %}
 <svg role="img" aria-label="Rating: 2.5/5" class="star-rating" focusable="false" height="12">
     <use xlink:href="#star-rating-2-5"></use>
 </svg>
     {% endhighlight %}
 
+    <h3>Method 2: Independent Stars</h3>
+    <p>This is built using fine-grained pieces (individual stars) for optimal flexibility and for allowing potential independant star treatments (i.e. animations).</p>
+    <p>
+        <ul class="demo-icon-boxes">
+            {% assign ratings = "0, 0-5, 1, 1-5, 2, 2-5, 3, 3-5, 4, 4-5, 5" | split: ", " %}
+            {% for rating in ratings %}
+            <li>
+                <span>{{ rating }}</span>
+                <div role="img" aria-label="Rating: 2.5/5" class="star-rating" data-stars="{{ rating }}" focusable="false">
+                    <svg class="star_rating_icon" aria-hidden="true">
```

Comment:
We typically don't use single underscores in BEM, only a double underscore, so this syntax of `star_rating_icon` needs reworking slightly.

We have two options, 1) use a double underscore to denote this element belongs under `star-rating`, e.g. `start-rating__icon` (but this might be weird as people could think it can also go under `svg.star-rating` - which it can't) or 2) start a new block with `star-rating-icon`. 

I'd be tempted to just go with 2) and start a new block. Which would then mean renaming it's elements to `star-rating-icon__symbol` and `star-rating-icon__star-half` too (we probably need to choose a better name than "symbol" there too, if we can).